<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nofile_dialog</class>
 <widget class="QDialog" name="nofile_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select a file</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select a valid file to analyze</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="ok_btn">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>If using the edit line, make sure to include the file path</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
